common/doc_store/gaussdb_conn_pool.py

```
@@ -0,0 +1,282 @@
+            if cur is not None:
+                cur.close()
+
+    def get_conn(self):
+        last_exc: Exception = RuntimeError("GaussDB connection checkout failed without an exception")
+        for _attempt in range(2):
+            conn = None
+            try:
+                conn = self._pool.getconn()
+                self._validate_conn(conn)
+                return conn
+            except Exception as exc:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)